### ISSUE TITLE
NPC's not using ranged weapons

### ISSUE THREAD

**Issue by jcd000** [CleverRaven/Cataclysm-DDA#15397]
Commit 0.C-10705-g16a9b72
(i think i built it during last week sometime. Friday 12/2 maybe?)

So.. no NPC has yet managed to use his ranged weapon against me... Faced one with a glock, another with a bow and another with a flamethrower... They turned hostile after me refusing to drop weapon.
I'd be dead if not for that bug..

**Comment by AdonaiJr** [comment]
Something very odd is happening these days... :P maybe is it related to the [#15390](https://github.com/CleverRaven/Cataclysm-DDA/issues/15390)?

**Comment by jcd000** [comment]
It is very likely that the new magazine system is to blame for all of this.. But bug report we have to.

**Comment by mugling** [comment]
Duplicate of #15088

**Comment by ejseto** [comment]
>Duplicate

They're not merely being confused by magazines. They refuse to shoot loaded unconverted weapons too. Last 2 hostile NPCs I ran into had an American 180 and a crossbow, both loaded when I finally killed them after getting tired of watching them stand around. Had the foresight to turn on debug mode the second time. When he was far away, he'd choose "melee" and walk towards me. Once he was in melee range, he'd choose "shoot" but do nothing.
